docs/how-do-i-install-my-operator-with-olm.md

```diff
@@ -28,23 +42,146 @@ spec:
   sourceNamespace: olm
  ``` 
 
-Once you have the subscription yaml, `kubectl apply -f Subscription.yaml` to install your operator. 
-
-You can inspect your Subscription with `kubectl get subs <name-of-your-subscription> -n <namespace>`.
+Once you have the YAML, `kubectl apply -f subscription.yaml` to install your operator. 
 
-To ensure the operator installed successfully, check for the ClusterServiceVersion and the operator deployment in the namespace it was installed in. 
+You can inspect your `Subscription` with `kubectl describe subs <name-of-your-subscription> -n <namespace>`.
 
+```yaml
+k describe subs subs-to-my-operator -n olm 
+Name:         subs-to-my-operator
+Namespace:    olm
+Labels:       <none>
+Annotations:  <none>
+API Version:  operators.coreos.com/v1alpha1
+Kind:         Subscription
+Metadata:
+  Creation Timestamp:  2019-11-19T13:54:51Z
+  Generation:          1
+  Resource Version:    1787
+  Self Link:           /apis/operators.coreos.com/v1alpha1/namespaces/operators/subscriptions/subs-to-my-operator
+  UID:                 2864cc19-0ad4-11ea-8c0e-aa8cc6bce8d3
+Spec:
+  Channel:           stable
+  Name:              my-operator
+  Source:            my-catalog
+  Source Namespace:  olm
+Status:
+  Catalog Health:
+    Catalog Source Ref:
+      API Version:       operators.coreos.com/v1alpha1
+      Kind:              CatalogSource
+      Name:              my-catalog
+      Namespace:         olm
+      Resource Version:  787
+      UID:               674c52e3-0ad2-11ea-8c0e-aa8cc6bce8d3
+    Healthy:             true
+    Last Updated:        2019-11-19T13:54:51Z
+  Conditions:
+    Last Transition Time:  2019-11-19T13:54:51Z
+    Message:               all available catalogsources are healthy
+    Reason:                AllCatalogSourcesHealthy
+    Status:                False
+    Type:                  CatalogSourcesUnhealthy
+  Current CSV:             my-operator.v1.15.0
+  Install Plan Ref:
+    API Version:       operators.coreos.com/v1alpha1
+    Kind:              InstallPlan
+    Name:              install-4xhbl
+    Namespace:         operators
+    Resource Version:  1744
+    UID:               286e5546-0ad4-11ea-8c0e-aa8cc6bce8d3
+  Installed CSV:       my-operator.v1.15.0
+  Installplan:
+    API Version:  operators.coreos.com/v1alpha1
+    Kind:         InstallPlan
+    Name:         install-4xhbl
+    Uuid:         286e5546-0ad4-11ea-8c0e-aa8cc6bce8d3
+  Last Updated:   2019-11-19T13:54:55Z
+  State:          AtLatestKnown
+Events:           <none>
 ```
-$ kubectl get csv -n <namespace-operator-was-installed-in>
+
+The `Subscription` has successfully deployed the Operator when the `status.state` property reaches `AtLatestKnown`.
```

Comment:
Is this always true? If the CSV reaches a `Failed` state, will the `Subscription` *not* reach `AtLatestKnow`?